packages/OmegaPrint-Tests.package/TestUI.class/instance/testFormatString.st

```diff
@@ -0,0 +1,14 @@
+button functionality
+testFormatString
+	
+	| cst evaluator |
+	cst := OhmExplicitSendsSmalltalk
+			match: 'testMethod'
+			startingFrom: #MethodDeclaration.
+	evaluator := OhmExplicitSendsSmalltalk synthesizedAttribute: OPPrinter new.
+	self 
+		assert: (OPPrinter
+					format: 'testMethod'
+					in: TestUI
+					notifying: nil)
+		equals: (evaluator value: cst)
```

Comment:
Das Ergebnis von einer Methode mit dem einer anderen zu vergleichen finde ich grenzwertig. Außerdem sollte man vielleicht zumindest noch ein Statement in den String packen, weil ein einzelnes Wort auch durch Zufall funktionieren könnte.